src/test/java/codeenthusiast/TrainingCenterApp/exercise/strengthexercise/keytechniqueelement/KeyTechniqueElementServiceImplTest.java

```diff
@@ -0,0 +1,39 @@
+package codeenthusiast.TrainingCenterApp.exercise.strengthexercise.keytechniqueelement;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+class KeyTechniqueElementServiceImplTest {
+
+    KeyTechniqueElement keyTechniqueElement1;
+    KeyTechniqueElement keyTechniqueElement2;
+    KeyTechniqueElement keyTechniqueElement3;
+
+    @BeforeEach
+    void setUp() {
+        keyTechniqueElement1 = new KeyTechniqueElement("Rozstaw nogi na szerkości barków");
+        keyTechniqueElement2 = new KeyTechniqueElement("Stopy delikatnie na zewnątrz");
+        keyTechniqueElement3 = new KeyTechniqueElement("Napnij mięsnie brzucha");
```

Comment:
We should choose one convention and use only english or only polish in test data fragments of code